src/App.vue

```diff
@@ -631,8 +616,8 @@ export default class App extends Mixins(CommonMixin, FilingTemplateMixin) {
     const email = this.getUserEmail
     const firstName = this.getUserFirstName
     const lastName = this.getUserLastName
-    // remove leading { and trailing } and tokenize string
-    const custom: any = { roles: this.getUserRoles?.slice(1, -1).split(',') }
```

Comment:
Background on this: user info _used to_ have a roles array, but that was removed at some point (without our knowledge). Anyway, getting it from KC token is safe, fast and reliable.